src/responder/kcm/secrets/secrets.c

```diff
@@ -856,58 +491,19 @@ errno_t sss_sec_new_req(TALLOC_CTX *mem_ctx,
     }
     req->sctx = sec_ctx;
 
-    ret = sss_sec_map_path(req, url, client, &req->mapped_path);
```

Comment:
Frankly, I'm quite confused with the amount of prepend/delete prefix (`SSS_SEC_BASEPATH` or `SSS_SEC_KCM_BASEPATH`) operations involved in this stuff.

Hopefully I didn't change format in my attempt to simplify this, but this place requires a careful look.

@sgoveas, do we have a test that checks that valid ccaches are still usable after sssd upgrade?

Reply:
Hi @alexey-tikhonov 
We don't have a test for it. I have created a task in Jira: SSSD-3830